src/scrollTo.js

```diff
@@ -143,6 +145,15 @@ export const scroller = () => {
     }
 
     container = _.$(options.container || defaults.container)
+
+    rtl = options.rtl || defaults.rtl
+
+    if (rtl) {
+      let childrens = [].slice.call(container.children)
+      let index = Array.prototype.indexOf.call(childrens.reverse(), element)
+      element = childrens.reverse()[index]
+    }
```

Comment:
Can you help me understand why this is needed? This seems like a potential performance bottleneck.

Perhaps we can improve this - just need to understand why it's needed!

Reply:
If direction reverse we should to scroll element that has reverse index

Reply:
How would we handle items that are not the same width?